style.css

```diff
@@ -0,0 +1,127 @@
+html {
+  -ms-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+
+body {
+  font-family: sans-serif;
+  line-height: 1.42857143;
+  color: rgb(238, 229, 229);
+  margin: 0;
+  padding: 0;
+}
+
+.flex-nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid black;
+  background-color: rgb(65, 60, 54);
+}
+
+.main-head {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 5px 10px;
+  padding: 5px 10px;
+  background-color: rgb(65, 60, 54);
+}
```

Comment:
- [ ] Kindly use this color: #3C3A39 for matching the same color as the Figma design on the header.
### your design
![your design](https://user-images.githubusercontent.com/81727988/170522909-78035e8b-56c2-420f-9ffb-2a24ccd1597e.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/170523041-2f61e79e-3730-4438-ad26-420ca62b6529.png)